### DISCUSSION TITLE
Chapters 11-13

### DISCUSSION THREAD

**Discussion by miscenko** [winitzki/sofp#33] [Q&A] [answered]
Hi Sergei,
Do I understad correctly that chapters 11-13 currently available only in printed on demand version at lulu?

Thanks,
Val

**Comment by winitzki** [comment, marked as answer]
Hi,
Sorry for late answer. No, chapters 11-13 are not yet finished. The entire material is available in pdf and at lulu.
Sergei

**Comment by winitzki** [comment]
@miscenko Chapters 11 and 12 are finished now. Only Chapter 13 remains unfinished, after that the book will be ready.
